### ISSUE TITLE
[Build] Implement a11y linting to detect violations

### ISSUE THREAD

**Issue by unlikelyzero** [nasa/openmct#7915]
#### Summary
https://github.com/vue-a11y/eslint-plugin-vuejs-accessibility

**Comment by evenstensberg** [comment]
Can I fix this?

**Comment by unlikelyzero** [comment]
> Can I fix this?

Yes. The ticket isn't fully fleshed out

**Comment by evenstensberg** [comment]
I've added a PR implementing the lint rules. After that is merged, you can turn each rule on and fix gradually.

**Comment by unlikelyzero** [comment]
@evenstensberg can you begin addressing the keyboard issues please

**Comment by evenstensberg** [comment]
@unlikelyzero Can we do that in seperate prs?

**Comment by unlikelyzero** [comment]
@evenstensberg please address one or two of the violations. Either the keyboard input or the other one

**Comment by evenstensberg** [comment]
Okay. I have exams this week + next, fyi. Will fix this when I got time :)
